src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10676,7 +10676,13 @@ private ExpressionSyntax ParseArrayOrObjectCreationExpression()
                         SyntaxFactory.MissingToken(SyntaxKind.CloseParenToken));
                 }
 
-                return _syntaxFactory.ObjectCreationExpression(@new, type, argumentList, initializer);
+                var objectCreation = _syntaxFactory.ObjectCreationExpression(@new, type, argumentList, initializer);
+                if (type.Kind == SyntaxKind.TupleType)
+                {
+                    objectCreation = this.AddError(objectCreation, objectCreation, ErrorCode.ERR_NewWithTupleTypeSyntax, type.ToString());
```

Comment:
By default we do attempt to bind syntax even if it has errors. We should not in his case. It is intentional that it means nothing now because we expect it to mean a different thing in the future.

---
In reply to: [81643102](https://github.com/dotnet/roslyn/pull/14161#discussion_r81643102) [](ancestors = 81643102)